Update README.md for legal and clarity (#15)
open-edge-platform/.github: profile/README.md

Author: ```suggestion
* [Edge AI Libraries](https://github.com/open-edge-platform/edge-ai-libraries) provide
  edge-optimized libraries, tools, SDKs, and microservices to support you in developing
  performant edge AI applications, along with user-friendly workflows for optimization
  and deployment.
* [Image Composer Tool](https://github.com/open-edge-platform/image-composer-tool) 
  is a framework providing a general-purpose toolchain for composing OS images from
  pre-built artifacts of any Linux distribution that supports Debian or RPM packages.
  You can generate edge-optimized OS images using Intel's pre-set YAML templates or your
  own customizations. 
* [Edge Microvisor Toolkit](https://github.com/open-edge-platform/edge-microvisor-toolkit) 
  is an edge-optimized container host offering the latest Intel silicon capabilities,
  for exploring advanced AI workloads at the edge. It is a minimal-footprint system
  based on the Azure Linux distribution, sharing its open source license.
* [Edge Workloads and Benchmarks](https://github.com/open-edge-platform/edge-workloads-and-benchmarks) 
  are performance-optimized pipelines that leverage the OpenVINO™ toolkit, the GStreamer
  multimedia framework, and Deep Learning Streamer (DL Streamer) for validating media
  and edge AI analytics.
```

# Welcome to the Open Edge Platform GitHub Organization! 

> Intel’s Open Edge Platform is a secure and optimized open platform for developing scalable edge solutions. 

Author: ```suggestion
Open Edge Platform is an Intel-optimized open software platform for developing and testing scalable edge solutions. 
```

> Open Edge Platform is an Intel-optimized open software platform for developing and testing scalable edge solutions. 

Reviewer: ```suggestion
Open Edge Platform is an Intel®-optimized open software platform for developing and testing scalable edge solutions. 
```

> It offers a modular, composable software stack that brings together the open source ecosystem to help you build performant edge solutions and securely operationalize them with cloud-like capabilities.

Author: ```suggestion
It offers a modular, composable software stack that brings together the open source ecosystem to help you design and build performant edge solutions that operate with cloud-like capabilities.
```
